service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/CafcassEmailContentProviderTest.java

```diff
@@ -2,30 +2,43 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import org.json.JSONObject;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.model.Orders;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
 import uk.gov.hmcts.reform.fpl.model.notify.submittedcase.SubmitCaseCafcassTemplate;
 import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+import uk.gov.hmcts.reform.fpl.utils.TestDataHelper;
 
 import java.util.List;
+import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.reform.fpl.enums.OrderType.CARE_ORDER;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
 import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
+import static uk.gov.hmcts.reform.fpl.utils.NotifyAttachedDocumentLinkHelper.generateAttachedDocumentLink;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
 
 @ContextConfiguration(classes = {CafcassEmailContentProvider.class, LookupTestConfig.class})
 class CafcassEmailContentProviderTest extends AbstractEmailContentProviderTest {
 
     @Autowired
     private CafcassEmailContentProvider cafcassEmailContentProvider;
 
+    private static final byte[] APPLICATION_BINARY = TestDataHelper.DOCUMENT_CONTENT;
+
     @Test
     void shouldReturnExpectedMapWithValidCaseDetails() {
+        final DocumentReference applicationDocument = testDocumentReference();
+        when(documentDownloadService.downloadDocument(applicationDocument.getBinaryUrl()))
```

Comment:
These 2 statements could be part of a before each and `applicationDocument` can then be a field instead